### PR TITLE
Reviewing project survey

### PR DESCRIPTION
**Pull request by jpol01**
Reviewing the code of the first project (Survey).

Play the role of the Code Reviewer by following some instructions:

-Follow the list from the article "How to give a code review?".

-Check if you have used semantic HTML tags.

-Check if you applied HTML & CSS best practices.

### CODE REVIEW

File: microverse3-trial-survey-main/index.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+            <input
+              name="user-recommend"
+              value="yes"
+              type="radio"
+              class="input-radio"
+              checked
+            />Yes
+          </label>
+
+          <!-- Case two: I DON'T THINK SO -->
```
Comment (jpol01) [inline microverse3-trial-survey-main/index.html:91]: Please keep capitalization basic, (in some cultures a text entirely in capital letters means that you are screaming, we know is a little mistake, kindly rewrite the phrase (don't think so) without all the capitals.

File: microverse3-trial-survey-main/index.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    <div class="container">
+      <!-- PAGE HEADER -->
+      <header class="header">
+        <h1 id="title" class="text-center">HAPPY SURVEY</h1>
+        <p id="description" class="description text-center">
+          Another happy survey, we really want to hear about you!!
+        </p>
+      </header>
+
+      <!-- PAGE FORM -->
```
Comment (jpol01) [inline microverse3-trial-survey-main/index.html:24]: Please keep capitalization basic, (in some cultures a text entirely in capital letters means that you are screaming, we know is a little mistake, kindly rewrite the phrase (page form) without all the capitals.

File: microverse3-trial-survey-main/index.html [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+          <label>
+            <input
+              name="user-recommend"
+              value="i dont think so"
+              type="radio"
+              class="input-radio"
+            />I dont think so
+          </label>
+
+          <!-- Case three:  NO -->
```
Comment (jpol01) [inline microverse3-trial-survey-main/index.html:101]: We know that is not your intention to be unrespectful so, please keep capitalization basic, (in some cultures a text entirely in capital letters means that you are screaming, we know is a little mistake, kindly rewrite the word  (no) without all the capitals.

**Review by jpol01 — COMMENTED** [review]
Great Job!! :fireworks:  Your Code looks clean, neat and easy to read is pretty well commented, which is awesome. :champagne: 

# Changes Required :recycle: 

Please check my comments below and do all the required changes.

Submit for a re-review when you are done :grin:

**Review by VanessaAoki — APPROVED** [review]
## STATUS: APPROVED 🟢 
Hi @jpol01, 

Your project is complete!  There is nothing else to say other than... it's time to merge it! 
Congratulations! 🎉 

approved

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 
- [x] Friendly written comments, explaining the reason behind each request 🔝 

#### For your next project, you should consider:
- [ ] There are some typos in your comments, in order to avoid those kinds of things, I would suggest that you double-check your texts with [Grammarly](https://www.grammarly.com/), they even have a Chrome extension!


Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jpol01** [comment]
Thanks for all your help Code Reviewers.

**Pull request merged**